docs/examples/pokeapi_process_multiple.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+import csv
+import time
+from typing import List
+
+from keboola.http_client import AsyncHttpClient
+
+
+def generate_jobs(nr_of_jobs):
+    return [{'method': 'GET', 'endpoint': str(endpoint)} for endpoint in range(1, nr_of_jobs+1)]
+
+def save_to_csv(results: List[dict]):
+    filename = "pokemon_details.csv"
+    fieldnames = ["name", "height", "weight"]  # Define the fields you want to store
+
+    with open(filename, "w", newline="") as csvfile:
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+        writer.writeheader()
+        for result in results:
+            writer.writerow({
+                "name": result["name"],
+                "height": result["height"],
+                "weight": result["weight"]
+            })
+
+async def main_async():
+    base_url = "https://pokeapi.co/api/v2/pokemon/"
+    start_time = time.time()
+
+    client = AsyncHttpClient(base_url=base_url, max_requests_per_second=20)
+
+    jobs = generate_jobs(1000)
+
+    results = await client.process_multiple(jobs)
```

Comment:
Is there a way to achieve this without having the `main_async` function `async`? E.g. something like [this](https://bitbucket.org/kds_consulting_team/kds-team.ex-chartmogul/src/53b565568c315cb4ed5cb167d8173945e00ad4e1/src/component.py?at=release%2FKCOFAC-797-optimise-customer_subscriptions#lines-78)  My idea was to have a function, that would run and wait for multiple jobs and just returned all the results (with the disadvantage of having to store all results in memory). Then I would just call something like `results = client.run_multiple_sync()` in my sync function. Is that possible?

Also I somehow dislike the name `process_multiple` what does it do? It actually sends multiple requests (not jobs) and returns list of result futures right? The request is basically the parameters of `_request_raw` http.py equivalent. I think it should be named something like `run_multiple_requests()`. Also I can imagine having equivalent for each method, e.g. `post_multiple`